bin/journey-test.sh

```diff
@@ -181,7 +181,7 @@ start_x_api() {
   pushd $xapi_home
 
   gem install bundler
-  bundle install
+  bundle install -j4
```

Comment:
It seemed to be a common option, but I guess I should have been more thoughtful about it :stuck_out_tongue:. [This blog](http://archlever.blogspot.com/2013/09/lies-damned-lies-and-truths-backed-by.html) indicates that the most effective amount of jobs is one less than the amount of cores, so I might add a function to the script to get the amount of cores and set the amount of jobs to one less